presets/4.3/tune/Tehllama_5in_2350-2750KV-Freedom_3S-4S-5S_Race_TUNE_Options.txt

```diff
@@ -0,0 +1,286 @@
+#$ TITLE: 5in TexasSpec Freedom Race Tune 2350-2750KV on 3S (Spec) and 4S/5S (Open)
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: 533, HighKV, FreedomSpec, Spec, Race, Llama, MultiVoltage
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: Built for EMULATING 533 FreedomSpec and similar builds on bidir DShot ESCs at 48kHz PWM with 23-27°/MedHigh timing
+#$ DESCRIPTION: 3S (2200mAh) will run a spec tune.  4S and 5S will run a conservative open race tune.
+#$ DISCLAIMER: This tune will auto-select profiles based on battery voltage at plugin, but only for 3S, 4S and 5S packs. Extensive testing has been done across 54+ builds, however not every craft will run best on this tune.
+#$ WARNING: Use at your own risk.  NO 6-cell tune or throttle limit is provided with this tune.  Plugging in a 6S pack with 3S profile active will likely cause damage.  
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/168
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# master
+set dshot_bidir = ON
+
+set dshot_idle_value = 440
+# Fallback value if dynamic idle does not work, or is deselected
+
+profile 0
+# profile 0 - 4S
+
+set vbat_sag_compensation = 77
+set iterm_rotation = ON
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set p_pitch = 56
+set i_pitch = 85
+set f_pitch = 176
+set p_roll = 49
+set i_roll = 74
+set d_roll = 36
+set f_roll = 153
+set p_yaw = 49
+set i_yaw = 74
+set f_yaw = 153
+set d_min_roll = 28
+set d_min_pitch = 35
+set d_max_advance = 0
+set auto_profile_cell_count = 4
+set feedforward_jitter_factor = 12
+set feedforward_boost = 12
+set throttle_boost = 12
+
+set simplified_master_multiplier = 95
+set simplified_i_gain = 85
+set simplified_pi_gain = 115
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 135
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+
+
+profile 1
+# profile 1 - 5S
+
+set vbat_sag_compensation = 77
+set iterm_rotation = ON
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set throttle_boost = 2
+set p_pitch = 42
+set i_pitch = 63
+set d_pitch = 34
+set f_pitch = 134
+set p_roll = 36
+set i_roll = 55
+set d_roll = 27
+set f_roll = 116
+set p_yaw = 36
+set i_yaw = 55
+set f_yaw = 116
+set d_min_roll = 21
+set d_min_pitch = 26
+set d_max_advance = 0
+set motor_output_limit = 92
+set auto_profile_cell_count = 5
+set feedforward_jitter_factor = 12
+set feedforward_boost = 8
+set throttle_boost = 5
+set feedforward_max_rate_limit = 103
+
+set simplified_master_multiplier = 65
+set simplified_i_gain = 85
+set simplified_d_gain = 110
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 150
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+set simplified_dterm_filter = OFF
```

Comment:
some filter changes in these presets are not covered with #$ include filter defaults. For example these ones.

You are modifying filter settings here, but you do not apply any filter defaults before that. So technically at this point you might end up with any random filters + your few filter changes.

Please check the same for the rest of the presets.